README.md

```diff
@@ -2,13 +2,13 @@
 CommunityFocus is a sharable time tracking app for teams.
 
 ## Getting Started
-To contibute:
+To contribute:
 
 1. Clone the backend and frontend repos (linked below).
 2. Install nodejs (we will be using npm).
 3. Open a terminal and `cd` into backend repo and run `npm i` and then `npm run dev`.
 4. Open another terminal and `cd` into the frontend repo. Run `npm i` and then `npm run dev`.
-5. Open your browser and go to `http://127.0.0.1:5173/` to open the app (temporary `http://127.0.0.1:5173/abc` to see timer).
+5. PORT number will default to 4000 unless specified in the '.env' file. Otherwise, open your browser and go to `http://127.0.0.1:4000/` to open the app (temporary `http://127.0.0.1:4000/abc` to see timer).
```

Comment:
```suggestion
5. PORT number will default to 4000 unless specified in the '.env' file as `PORT`. Otherwise, open your browser and go to `http://localhost:4000/` to open the backend server.
6. However, you will need the frontend repo to run the complete app. Clone and run https://github.com/CommunityFocus/cf-frontend to see the frontend.
```

There's no need for the "temporarily" anymore since the user can click on the `join room` button.

Also updated the localhost IP to `localhost` instead of `127.0.0.1`